packages/bot-web-ui/src/utils/hotjar.js

```diff
@@ -0,0 +1,33 @@
+import { epochToMoment, toMoment } from '@deriv/shared';
+
+const hotjar = client => {
+    /**
+     * Inject: External Script Hotjar - for DBot only
+     */
+    (function (h, o, t, j) {
+        /* eslint-disable */
+        h.hj =
+            h.hj ||
+            function () {
+                (h.hj.q = h.hj.q || []).push(arguments);
+            };
+        /* eslint-enable */
+        h._hjSettings = { hjid: 3050531, hjsv: 6 };
+        const a = o.getElementsByTagName('head')[0];
+        const r = o.createElement('script');
+        r.async = 1;
+        r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
+        a.appendChild(r);
+
+        const user_id = client.loginid;
```

Comment:
is optional check not required client?.loginid. when user is not logged in ?

Reply:
@vinu-deriv Yes we can add optional here. But need to check how hotjar provides the data. Thanks.